source/time_tables/departure_boards.cpp

```diff
@@ -65,6 +66,18 @@ static bool is_terminus_for_all_stop_times(const std::vector<routing::datetime_s
     return !stop_times.empty();
 }
 
+void update_display_information(bool& vj_found,
```

Comment:
you can return instead of passing a non const ref

```suggestion
bool update_display_information(
```